R/04-miRNAPrioritization.R

```diff
@@ -30,175 +30,185 @@
+
+      if (m %in% c("AggInv", "AggLog")) {
+        # perform sampling
+        sampling.data <- samplingDataBase(enrich.null,
+          m.selector,
+          samp.rate,
+          fn,
+          n_paths,
+          sampling.data.file,
+          jack.knife = FALSE,
+          save.sampling = save.sampling,
+          num.cores = 8
+        )
+
+        m.selector <- methodProbBase(
+          sampling.data = sampling.data[[paste0("SampSize_", 100)]],
+          selector = m.selector,
+          m = m,
+          n_paths = n_paths,
+          cover.fn = cover.fn
+        )
+      } else {
+        names(m.selector)[2:3] <- paste0(m, "_", names(m.selector)[2:3])
       }
-     
-      
+
+
       print(paste0(m, " Method Done"))
-      
+
       # perform jack-knife
-      if (run.jack.knife ==T){
+      if (run.jack.knife == T) {
         # jack.knife.data.filename  <- paste0(prefix, m, "_", samp.rate, "_samples_jack_knifed.RDS")
         # jack.knife.data.file <- paste0(sampling.data.dir, '/', jack.knife.data.filename)
-        # 
-        # jack.knife.data <- sampling.data.base(enrich.null, m.selector, samp.rate, fn, n_paths, 
+        #
+        # jack.knife.data <- sampling.data.base(enrich.null, m.selector, samp.rate, fn, n_paths,
         #                                       sampling.data.file=jack.knife.data.file, jack.knife=TRUE, save.sampling = save.jack.knife)
-        # 
-        
+        #
+
         sampling.data <- samplingDataBase(enrich.null,
-                                             m.selector,
-                                             samp.rate,
-                                             fn,
-                                             n_paths,
-                                             sampling.data.file,
-                                             jack.knife=FALSE,
-                                             save.sampling = save.sampling,
-                                             num.cores = 8)
-        m.selector <- jackKnifeBase(selector    = m.selector,
-                                       pathways    = pathways,
-                                       enrich.null = enrich.null,
-                                       fn = fn,
-                                       jack.knife.data = 
-                                         sampling.data[[paste0("SampSize_",100)]],
-                                       m  = m, 
-                                       num.cores = 8)
-        
+          m.selector,
+          samp.rate,
+          fn,
+          n_paths,
+          sampling.data.file,
+          jack.knife = FALSE,
+          save.sampling = save.sampling,
+          num.cores = 8
+        )
+        m.selector <- jackKnifeBase(
+          selector = m.selector,
+          pathways = pathways,
+          enrich.null = enrich.null,
+          fn = fn,
+          jack.knife.data =
+            sampling.data[[paste0("SampSize_", 100)]],
```

Comment:
This can fit on one line now?

Reply:
The styler package does some magic indeed!

Reply:
Yes but it can't fix everything so it still needs some help.